spec/spec_helper.rb

```diff
@@ -5,8 +5,14 @@
 require 'fakeweb'
 require 'pry'
 require 'rspec'
+
 require 'simplecov'
-SimpleCov.start
+require 'coveralls' if ENV['CI']
+
+SimpleCov.formatter = Coveralls::SimpleCov::Formatter if ENV['CI']
+SimpleCov.start do
+  add_filter '/spec/'
+end
```

Comment:
Hm. Do you know what benefit the profiles gives us, if we're not sharing them across multiple projects?

Good tip about the `MultiFormatter`, and I see they're now encouraging that in the coveralls documentation.